build.gradle

```diff
@@ -50,7 +50,7 @@ repositories {
 esplugin {
     name project.properties['pluginName']
     description 'Safely expose Elasticsearch REST API'
-    classname 'org.elasticsearch.plugin.readonlyrest.wiring.ReadonlyRestPlugin'
+    classname 'org.elasticsearch.plugin.readonlyrest.es53x.ReadonlyRestPlugin'
```

Comment:
If I understood correctly, when I will have implemented the equivalent package for `es52x`, I will just need to change this line and `bin/build.sh` again, right?